command/harness/destroy.go

```diff
@@ -3,14 +3,20 @@ package harness
 import (
 	"context"
 	"fmt"
+	"time"
 
+	"github.com/cenkalti/backoff/v4"
 	"github.com/drone-runners/drone-runner-aws/internal/drivers"
 	ierrors "github.com/drone-runners/drone-runner-aws/internal/types"
 	"github.com/drone-runners/drone-runner-aws/store"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
+var (
+	destroyTimeout = 10 * time.Minute
```

Comment:
I think it would be better to pass the value in, than declared at the top? Also is the value to high?

Reply:
This value is the max time an init task can be stuck. Currently the values are defined in the runner since it's doing the retries itself instead of being retried from the manager framework: https://github.com/drone-runners/drone-runner-aws/blob/master/command/harness/dlite/init.go#L16